modules/openy_features/openy_prgf/openy_prgf.info.yml

```diff
@@ -11,5 +11,6 @@ dependencies:
   - drupal:options
   - drupal:taxonomy
   - drupal:text
+  - entity_clone
```

Comment:
Can we move `entity_clone.settings.yml` to optional configs and remove this dependency?

Reply:
+ we should import it by using code, not by using module installation routine. Configs, named by module's name conflicting